tsfmt.json

```diff
@@ -10,6 +10,7 @@
 	"insertSpaceAfterFunctionKeywordForAnonymousFunctions": true,
 	"insertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis": false,
 	"insertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets": false,
+	"insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": true,
```

Comment:
I'm inclined to leave this file alone for now since it is used for the VSCode codebase itself. Any specific reasons for adding this setting?